File: vet-management-ui/src/app/models/treatmentItem.ts
import { Medication } from "./medication";
import { Service } from "./service";

export interface TreatmentItem {
    treatmentItemId: number;
    medication?: Medication;
    service?: Service;
    nameAtTreatment: string;
    unitPrice: number;
    quantity: number;
    reason: string;
}

File: VetManagement/Models/TreatmentItem.cs
namespace VetManagement.Models
{
    public class TreatmentItem
    {
        public int TreatmentItemId { get; set; }
        public int TreatmentId { get; set; }
        public int? MedicationId { get; set; }
        public Medication? Medication { get; set; }
        public int? ServiceId { get; set; }
        public Service? Service { get; set; }
        public string NameAtTreatment { get; set; } = String.Empty;
        public decimal UnitPrice { get; set; }
        public int Quantity { get; set; }
        public string Reason { get; set; } = String.Empty;
    }
}
